lib/utils/messages.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+'use strict';
+
+import { MessageType, MessageLevel } from '../../types/types';
+
+
+const GREEN = '\x1B[32m';
+const YELLOW = '\x1b[33m';
+const RED = '\x1B[31m';
+const RESET = '\x1B[0m';
+const BOLD = '\x1b[1m';
+
+const greenify: ((str: string) => string) = str => `${GREEN}${str}${RESET}`;
+const redify: ((str: string) => string) = str => `${RED}${str}${RESET}`;
+const yellowify: ((str: string) => string) = str => `${YELLOW}${str}${RESET}`;
+const boldify: ((str: string) => string) = str => `${BOLD}${str}${RESET}`;
+
+const getMessage = function(messageType: MessageType, ...args: string[]) {
```

Comment:
I think we can use `string` instead of `MessageType`, just make it simpler, not to have here and in types.